packages/integration-tests/index.js

```diff
@@ -0,0 +1,172 @@
+
+/**
+ * Wait for the defined number of milliseconds
+ *
+ * @param {integer} waitPeriod - number of milliseconds to wait
+ * @returns {Promise.<undefined>} - promise resolves after a given time period
+ */
+function timeout(waitPeriod) {
+  return new Promise((resolve) => setTimeout(resolve, waitPeriod));
+}
+
+/**
+ * Get the list of workflows for the stack
+ *
+ * @param {string} stackName - Cloud formation stack name
+ * @param {string} bucketName - S3 internal bucket name
+ * @returns {Object} list as a JSON object
+ */
+async function getWorkflowList(stackName, bucketName) {
+  const key = `${stackName}/workflows/list.json`;
+
+  return s3().getObject({ Bucket: bucketName, Key: key }).promise()
+    .then((workflowJson) => JSON.parse(workflowJson.Body.toString()));
+}
+
+/**
+ * Get the template JSON from S3 for the workflow
+ *
+ * @param {string} stackName - Cloud formation stack name
+ * @param {string} bucketName - S3 internal bucket name
+ * @param {string} workflowName - workflow name
+ * @returns {Object} template as a JSON object
+ */
+async function getWorkflowTemplate(stackName, bucketName, workflowName) {
+  const key = `${stackName}/workflows/${workflowName}.json`;
+  return s3().getObject({ Bucket: bucketName, Key: key }).promise()
+    .then((templateJson) => JSON.parse(templateJson.Body.toString()));
+}
+
+/**
+ * Get the workflow ARN for the given workflow from the
+ * template stored on S3
+ *
+ * @param {string} stackName - Cloud formation stack name
+ * @param {string} bucketName - S3 internal bucket name
+ * @param {string} workflowName - workflow name
+ * @returns {string} - workflow arn
+ */
+async function getWorkflowArn(stackName, bucketName, workflowName) {
+  return getWorkflowTemplate(stackName, bucketName, workflowName)
+   .then((template) => template.cumulus_meta.state_machine);
+}
+
+/**
+ * Get the execution status (i.e. running, completed, etc)
+ * for the given execution
+ *
+ * @param {string} executionArn - ARN of the execution
+ * @returns {string} status
+ */
+async function getExecutionStatus(executionArn) {
+  return sfn().describeExecution({ executionArn }).promise()
+    .then((status) => status.status);
+}
+
+/**
+ * Wait for a given execution to complete, then return the status
+ *
+ * @param {string} executionArn - ARN of the execution
+ * @returns {string} status
+ */
+async function waitForCompletedExecution(executionArn) {
+  let executionStatus = await getExecutionStatus(executionArn);
+  let statusCheckCount = 0;
+
+  // While execution is running, check status on a time interval
+  while (executionStatus === 'RUNNING' && statusCheckCount < executionStatusNumRetries) {
+    await timeout(waitPeriodMs);
+    executionStatus = await getExecutionStatus(executionArn);
+    statusCheckCount++;
+  }
+
+  if (executionStatus === 'RUNNING' && statusCheckCount === executionStatusNumRetries) {
+    console.log(`Execution status check timed out, exceeded ${executionStatusNumRetries} status checks.`);
```

Comment:
Add ` // eslint-disable-line max-len` at the end of this line so it doesn't complain that the line is too long.  In the case of long strings, exceeding the max line length isn't a problem.

http://airbnb.io/javascript/#strings--line-length